package/yast2-squid.spec

```diff
@@ -39,7 +39,6 @@ BuildRequires:  update-desktop-files
 BuildRequires:  yast2 >= 4.1.0
 BuildRequires:  yast2-core-devel
 BuildRequires:  yast2-devtools >= 4.2.2
-BuildRequires:  yast2-testsuite
```

Comment:
I think also perl-XML-Writer can be removed.

Reply:
Done.